templates/zerver/integrations/discourse.md

```diff
@@ -1,16 +1,39 @@
-1.  {!create-an-incoming-webhook.md!}
+# Zulip Discourse integration
 
-    The API keys for "Incoming webhook" bots are limited to only
-    sending messages via webhooks. Thus, this bot type lessens
-    the security risks associated with exposing the bot's API
-    key to third-party services.
+Get Discourse notifications in Zulip!
 
-1.  Copy the `USERNAME` and `API KEY` - you'll need it later.
+{start_tabs}
 
-1.  Head over to the
-    [discourse-chat-integration setup instructions](https://meta.discourse.org/t/68501)
-    and complete them.
+1. {!create-an-incoming-webhook.md!}
+
+1. Install the Discourse [Chat Integration][chat-integration].
```

Comment:
I've re-written the Discourse integration instructions. Instead of redirecting to their site, I've moved the instructions to our page to conform to our formatting.
As a follow-up, I can post a comment in their page, requesting them to update their Zulip-specific instructions to point to our page instead.